src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
```

Comment:
@joshcurtis @s-gogna 
Should we keep all the `defremote` in `client_interop.clj`?
I feel like it involves too much code repitition